packages/expo-font/src/__tests__/Font-test.ts

```diff
@@ -17,286 +27,322 @@ afterEach(() => {
   jest.resetModules();
 });
 
-describe('loadAsync', () => {
-  it(`completes after loading a font`, async () => {
-    const NativeFontLoader = NativeModulesProxy.ExpoFontLoader;
+describe('within expo client', () => {
+  beforeAll(() => {
+    jest.mock('expo-constants', () => ({
```

Comment:
IIRC `mock` is hoisted to run before `imports`(specifically, the `requires` created from `imports`) while `doMock` isn't. So `mock` is correct only at the top-level.

If the tests fail with `doMock`, probably what needs to happen is to call `resetModules` and re-import them in each `describe` block.

```suggestion
    jest.doMock('expo-constants', () => ({
```

Also, move this into the `describe` outside of `beforeAll` since the constants aren't mutated between tests in the describe block.